pyvista/examples/downloads.py

```diff
@@ -3578,81 +3580,34 @@ def download_action_figure(load=True):  # pragma: no cover
     return _download_and_read('tigerfighter.obj', load=load)
 
 
-def download_mars_jpg():  # pragma: no cover
+def download_mars_jpg():
```

Comment:
Instead of the `_jpg`, etc suffixes, could we use `_texture` to signify that they are loading as `pyvista.Texture` objects?

Any methods added here should do that. Then in a follow up PR, we can deprecate other, existing methods that load textures.

cc @adeak

Reply:
Thanks for the comment. It was resolved in https://github.com/pyvista/pyvista/pull/2994/commits/28ee70337b06ea90d2185834f94c3a8d1bc38f5d .